public/app.js

```diff
@@ -0,0 +1,101 @@
+var app = angular.module("superCrudApp", ["ngRoute", "ngResource"]);
+
+
+// factory
+
+app.factory("Books", ["$http", function($http) {
+	var baseUrl = "https://api.parse.com/1/classes/Book/";
+    function getParams() {
+    	return {
+    		headers: {
+	         'X-Parse-Application-Id': '927W7TTGno6KNkUCJ7pGh8iHVgXUh7qyL7wnrHfT',
+	         'X-Parse-REST-API-Key': '9VJU7EW7WKcu0H2srbwScyX8qJLvNFH4OlrDaEr0'
+	    	}
+	    };
+    }
+	return {
+		query:function(){
+			return $http.get(baseUrl,getParams());
+		},
+		get:function(id){
+			return $http.get(baseUrl+id,getParams());
```

Comment:
You could improve this using Angular's `$resource` service but this is nicely done.